### ISSUE TITLE
Update for UE from Source 5.3.2 ??

### ISSUE THREAD

**Issue by jittdev** [Incurian/VR_Template_UE5.1_SB#1]
Hi Incurian. Watched your videos on this at https://forums.unrealengine.com/t/free-vr-template-blueprints-only-multiplayer-ue-5/1185920. Am intrigued and want to try to get this working for Meta Quest 3 Dev.

However, the forked project won't build with my from source UE5.3.2 (which is stable), as it changes (reverts) too many UE dlls etc. I know it may be asking a lot, but is there any way you can update this project to work with UE5.3.2 from source (specifically the Meta Quest/Oculus branch)?

**Comment by Incurian** [comment]
I can try. Send me the link to the exact engine source you want me to use.
Does opening the template in 5.1 and then migrating the content folder to
an existing 5.3 project not work? Discord is better for me, I almost didn't
see this email, can you join there? https://discord.com/invite/ANt8bbh3rA

**Comment by jittdev** [comment]
Sure, I'll join...but just in case to answer above: I migrated the NewVRTemplate only into a new project (without starter content). But still run into two unresolved compiling errors in Pawn_VR_22 blueprint and also EnterMap. These might be easy fixes for you, but as I am still familiarizing myself with your logic....

Pawn_VR_22 error: Pin for variable "Down_Text" recreated, but the variable is missing.
EnterMap event graph error : Do Call Function by Name has somehow lost its actual function call

(also: I opened the project in 5.2 successfully (with UE regular build from EpicGames launcher, not Oculus fork) then migrated to the 5.3 Oculus fork)
